doc/install/install-ceph-gateway.rst

```diff
@@ -234,12 +234,11 @@ Configure Bucket Sharding
 -------------------------
 
 A Ceph Object Gateway stores bucket index data in the ``index_pool``, which
-defaults to ``.rgw.buckets.index``. Sometimes users like to put many objects
-(hundreds of thousands to millions of objects) in a single bucket. If you do
-not use the gateway administration interface to set quotas for the maximum
-number of objects per bucket, the bucket index can suffer significant
-performance degradation when users place large numbers of objects into a
-bucket.
+defaults to ``.rgw.buckets.index``. Sometimes, users like to put many objects
+(hundreds of thousands to millions of objects) in a single bucket. If you did
+not set quotas for the maximum number of objects per bucket using the gateway
+administration interface, the bucket index can suffer significant
+performance degradation.
```

Comment:
i don't think "when users place large numbers of objects into a bucket" is redundant here. you could put 

> In this case, if you do not ....

to connect the precondition to its consequence.